[FEATURE] Changer le format de date en NL sur la page de réconciliation (PIX-13985)

pix-service-prod opened the pull request:
Il y avait une incohérence dans les formats de date sur la page de réconciliation SCO-FWB : 
![image (18)](https://github.com/user-attachments/assets/09e16136-4a30-467f-b1f5-3f689f9dbb21)

En NL on doit avoir la date au format suivant : dd-mm-jjjj

pix-service-prod committed:
Updated translations from Phrase

phrase: update mon-pix/translations/nl.json to latest state
phrase: update mon-pix/translations/es.json to latest state

pix-bot-github commented:
Une fois les applications déployées, elles seront accessibles via les liens suivants :
  * [App (.fr)](https://app-pr9949.review.pix.fr)
  * [App (.org)](https://app-pr9949.review.pix.org)
  * [Orga (.fr)](https://orga-pr9949.review.pix.fr)
  * [Orga (.org)](https://orga-pr9949.review.pix.org)
  * [Certif (.fr)](https://certif-pr9949.review.pix.fr)
  * [Certif (.org)](https://certif-pr9949.review.pix.org)
  * [Junior](https://junior-pr9949.review.pix.fr)
  * [Admin](https://admin-pr9949.review.pix.fr)
  * [API](https://api-pr9949.review.pix.fr/api/)
  * [Audit Logger](https://pix-audit-logger-review-pr9949.osc-fr1.scalingo.io/api/)

Les variables d'environnement seront accessibles via les liens suivants :
  * [scalingo front](https://dashboard.scalingo.com/apps/osc-fr1/pix-front-review-pr9949/environment)
  * [scalingo api](https://dashboard.scalingo.com/apps/osc-fr1/pix-api-review-pr9949/environment)
  * [scalingo audit-logger](https://dashboard.scalingo.com/apps/osc-fr1/pix-audit-logger-review-pr9949/environment)

pix-service-prod committed:
Updated translations from Phrase

phrase: update mon-pix/translations/nl.json to latest state
phrase: update mon-pix/translations/es.json to latest state

pix-service-prod committed:
Updated translations from Phrase

phrase: update mon-pix/translations/nl.json to latest state
phrase: update mon-pix/translations/es.json to latest state

machestla reviewed (approved):
Tech review Ok ✅
